content/docs/fuzzing/c-cpp/11-aflpp/index.md

```diff
@@ -438,11 +443,14 @@ You will see that the execution speed is significantly slower compared to the fu
 
 The above example no longer uses persistent mode, because we switched away from the libFuzzer harness. However, the persistent mode can be re-enabled, as shown in the next section.
 
-### Optimizing the fuzzer: Enable persistent mode {#optimizing-the-fuzzer-enable-persistent-mode}
+### Optimizing the fuzzer: Enable persistent mode & shared memory {#optimizing-the-fuzzer-enable-persistent-mode}
+
+Enabling persistent mode improves fuzzing performance by a factor of 10 to 20 (see the above tip for details about why persistent mode is faster). Note that the initial version we started with already runs in persistent mode because it uses a libFuzzer-style harness. 
+We will also enabled shared memory for the fuzzing campaign which means that test cases are passed from the fuzzer to the SUT via shared memory instead of using standard input, arguments or reading from files.
```

Comment:
```suggestion
We will also enable shared memory for the fuzzing campaign, which means that test cases are passed from the fuzzer to the SUT via shared memory instead of using standard input, arguments, or reading from files.
```